cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/SpaceDevelopersOperations.java

```diff
@@ -60,111 +46,54 @@ public SpaceDevelopersOperations(DefaultCloudFoundryOperations cfOperations) {
     }
 
     /**
-     * Assign a user as a space developer
+     * Prepares a request for the space id to the cf instance.
+     * The space id is needed for assigning/removing space developers.
+     * @return Mono object which yields the space id upon subscription
+     */
+    public Mono<String> getSpaceId() {
+        return cloudFoundryOperations.getSpaceId();
```

Comment:
I think the method helps to increase the cohesion, because the spaceId is required in order to manipulate the space developers. Not everyone might be familiar with querying the spaceId directly from the cf operations and in my opinion we should always try to get rid of any request building in the controller classes and move it to the operation classes.